### PR TITLE
Run as a non root user

### PR DESCRIPTION
**Pull request by easternbloc**
This isn't tested but I think it's all that's required to get the container running as a non root user.

Let me know if I've missed anything.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,12 @@ RUN apk update && \
 
 ADD templates/ opt/templates
 ADD bin/keycloak-proxy /opt/keycloak-proxy
-RUN chmod +x /opt/keycloak-proxy
+RUN useradd keycloak && \
```
Comment (gambol99) [inline Dockerfile]: It's alpine linux, you would probably need something like 
```shell
addgroup -S keycloak
adduser -G keycloak -S keycloak
```

File: Dockerfile [hunk 2]
```diff
@@ -6,8 +6,12 @@ RUN apk update && \
...
+    chmod +x /opt/keycloak-proxy && \
+    chown keycloak:keycloak /opt/keycloak-proxy
```
Comment (gambol99) [inline Dockerfile]: this line shouldn't be required, as it should already been executable

**Comment by easternbloc** [comment]
@gambol99 I notice you changed all the things since I did this. Let me know if you're happy with that approach!

**Comment by easternbloc** [comment]
@gambol99 pingy pong!

**Pull request closed without merging**